.github/pull_request_template.md

```diff
@@ -1,24 +1,39 @@
-Describe what the changes are meant to address. If the implementation is non trivial, please also include a short overview.
+**Select one or more of:**
 
+<!-- To check an item, fill the brackets with the letter x; the result should look like `[x]`. -->
 
+- [ ] Issue
+- [ ] Feature
+- [ ] Fix
+- [ ] Housekeeping
+- [ ] Other:
+
+---
+
+**Description of PR**
+
+<!-- This should include a description of the bug/feature and what you added/changes/removed -->
+
+[insert text here]
 
 ---
 
 <!-- Tags (fill and keep as many as applicable): -->
 
 Fixes: #number of issue (implies Closes tag) or commit SHA
+
 Closes: #number of issue or pull request
```

Comment:
Can this be moved to being first, I find it odd that its after the description. 

IMO the issue it closes I would expect to be first.

Reply:
Yup, I agree, moved it to first in my latest push